plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintExtension.kt

```diff
@@ -29,11 +35,14 @@ open class KtlintExtension {
     var ignoreFailures = false
 
     /**
-     * Report output format.
+     * Report output formats.
      *
      * Available values: plain, plain_group_by_file, checkstyle, json.
      *
-     * Default is plain.
+     * Default is empty.
      */
-    var reporter: ReporterType = ReporterType.PLAIN
+    var reporters: Array<ReporterType> = arrayOf(ReporterType.plain)
+
+    @Deprecated(message = "Ktlint introduced multi output support since 0.11.1 version", replaceWith = ReplaceWith("reporters = arrayOf(x)", "reporter"))
```

Comment:
Seems `reporters = arrayOf(x)` not working as expected - it produces `reporters = arrayOf(x)`.

Change it to `ReplaceWith("reporters = arrayOf()")`